### DISCUSSION TITLE
Roadmap for v1.3

### DISCUSSION THREAD

**Discussion by w-ahmad** [w-ahmad/WinUI.TableView#39] [Feedback Wanted]
A sneak peek at the upcoming changes, features, and improvements in WinUI.TableView v1.3.0 is here:

- Presentation
  - Grid lines support
  - Alternate row color support
  - Better customization support
  - Localization support

- New Functionalities
  - Support for custom filter and sort implementation
  - Row context menu

- New column types
  - Date column (supports DateTimeUtc, DateTime, and DateOnly)
  - Time column (supports DateTimeUtc, DateTime, TimeOnly and TimeSpan)
  - CommandBar column (for row commands)

**Comment by RossCoeH** [comment]
I like your project
I think you should reconsider sorting options to become just a click creating first ascending sort, 
next click creates descending, 
next click removes sort.
 It seemed counterintuitive to have only ascend, descend,,then needing a right click to clear sort.
I think there is still a place for right click to create custom sort, but probably the nextclick on sort icon should clear custom sort.

**Reply by w-ahmad** [reply]
Thank you @RossCoeH for the suggestions. A similar enhancement #37 is already requested and would be available in [v1.3.0](https://github.com/w-ahmad/WinUI.TableView/milestone/1)

**Comment by w-ahmad** [comment]
Checkout the milestone [v1.3.0](https://github.com/w-ahmad/WinUI.TableView/milestone/1) to see which issues are planned for v1.3. A few of them are not listed above.

**Reply by JLBLU** [reply]
Hi Ahmad
Friendly and useful. 
Can you add/check property "Tag" in next release ? (cannot be set in TableViewTextColumn , TableViewTemplateColumn ...)
Thank you.

**Reply by w-ahmad** [reply]
Thank you @JLBLU for your suggestion, I don't see any reason why Tag property can't be added to the column. I'll make sure it's there in next release.

**Reply by JLBLU** [reply]
Thank you and sorry, I saw your response late. So, I recreated a post wanting to clarify my needs. This last post is therefore useless (but the need for a tag is very useful). So, ignore the last discussion #49.
Post-Scriptum: in the meantime, I have overloaded your classes to add <tag> management (TableView,TableViewColumnsCollection,TableViewColumn,TableViewColumnPropertyChanged,TableViewBoundColumn
). It works, but it will work even better when it's native ... Thanks.

**Reply by JLBLU** [reply]
Hi Ahmad
On TableViewTextColumn, CTRL/C CTRL/V (keyboard keys) not working properly when clicking in cell (it works using right click).
Would it be also possible, in a future release, to add some  events on columns : GettingFocus, GotFocus ? Thanks

**Reply by w-ahmad** [reply]
Hi @JLBLU, I have added the Tag property in #84 and will try to address potential issues with Ctrl+C and Ctrl+V in a future release. Please consider closing discussion #49, if it is not more needed. Thanks.

**Comment by w-ahmad** [comment]
🚀 First preview of v.1.3.0 is live!

https://github.com/w-ahmad/WinUI.TableView/releases/tag/v1.3.0-preview1

**Comment by w-ahmad** [comment]
v1.3.0 is out now!
https://github.com/w-ahmad/WinUI.TableView/releases/tag/v1.3.0
